mautic/mautic
# [UI] Reset company email column style

| Q                                      | A
| -------------------------------------- | ---
| Bug fix? (use the a.b branch)          | 🟢 
| New feature/enhancement? (use the a.x branch)      | 🔴
| Deprecations?                          | 🔴
| BC breaks? (use the c.x branch)        | 🔴
| Automated tests included?              | 🔴 
| Related user documentation PR URL      | mautic/user-documentation#... 
| Related developer documentation PR URL | mautic/developer-documentation-new#... 
| Issue(s) addressed                     | Fixes #... 



## Description

This pull request addresses a visual inconsistency in the company email column by removing the "small" class from the email cell in the list row. Previously, the email text appeared smaller than intended, which could lead to readability issues for users.

By standardizing the text size, we enhance the overall user experience, making it easier for users to scan and identify email addresses quickly. This change benefits all users interacting with the list, ensuring that important contact information is displayed clearly and consistently.




| Before                                 | After
| -------------------------------------- | ---
|  ![image](https://github.com/user-attachments/assets/861b0bb1-e7ae-438f-9129-60772369fe45)  |  ![image](https://github.com/user-attachments/assets/9f9820ca-811a-482f-bfc0-c7c05c7f1a0f)                                    



---
### 📋 Steps to test this PR:


1. Open this PR on Gitpod or pull down for testing locally (see docs on testing PRs [here](https://contribute.mautic.org/contributing-to-mautic/tester))
2. Open the Companies page, create a new company filling the email, save it, and then close it and go back to the list to see the table.
